tests/test_distributedarray.py

```diff
@@ -0,0 +1,101 @@
+"""Test the DistributedArray class
+    Designed to run with n processes
+    $ mpiexec -n 10 pytest -m mpi test_distributedarray.py
```

Comment:
```
  /home/hongy0a/pylops-mpi/tests/test_distributedarray.py:27: PytestUnknownMarkWarning: Unknown pytest.mark.mpi - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.mpi(min_size=2) 
```
currently I got this warning when launching this test file.

Reply:
I think you do not have pytest-mpi installed
just do a `pip install pytest-mpi` or just install all the requirements using `pip install -r requirements-dev.txt`
Then either use `make tests` or use `mpiexec -n 10 pytest test_distributedarray.py --with-mpi`
Then it will work accordingly...

Reply:
I had done the same mistake initially (using an old env). I confirm this works :)